docs/querying/sql.md

```diff
@@ -877,7 +877,7 @@ Submit your query as the value of a "query" field in the JSON object within the
 |--------|----|-----------|
 |`query`|SQL query string.| none (required)|
 |`resultFormat`|Format of query results. See [Responses](#responses) for details.|`"object"`|
-|`header`|Whether or not to include a header. See [Responses](#responses) for details.|`false`|
+|`header`|Whether or not to include a header for the query response. See [Responses](#responses) for details.|`false`|
```

Comment:
wait. I think there is some confusion here. The `header` here is talking about the column names of each column in the result. This is different for the header of the HTTP response.

Reply:
@maytasm Yeah, that's what I was trying to differentiate but not very well. Maybe "query result" would be less confusing?